lib/backburner/deferred-action-queues.js

```diff
@@ -18,6 +18,10 @@ function noSuchQueue(name) {
   throw new Error('You attempted to schedule an action in a queue (' + name + ') that doesn\'t exist');
 }
 
+function noSuchMethod(name) {
+  throw new Error('You attempted to schedule an action in a queue (' + name + ') for a method that doesn\'t exist');
```

Comment:
Yep, I'm not entirely happy with it, but currently lack the inspiration for a better one. Do you have a suggestion?